src/components/standardForm/StandardFormSectionDescription.module.css

```diff
@@ -3,5 +3,4 @@
     font-size: 14px;
     line-height: 19px;
     color: var(--colors-grey800);
-    max-width: 600px;
```

Comment:
I've removed this as the description text for program indicator is quite long. I don't think this is really useful as the text is a <p> element, and we probably want to let it scroll if needed. In some forms, e.g. open Data Element, maybe you would not want the description to scroll beyond the input width (which could happen in translation), but I think if we have a real reason for that, we should put the form in a div with restricted width, rather than restricting the width on this one component.